terraform/projects/infra-security-groups/main.tf

```diff
@@ -151,16 +151,7 @@ resource "aws_security_group_rule" "monitoring_external_sg_ingress_any_https" {
   from_port         = 443
   protocol          = "tcp"
   security_group_id = "${aws_security_group.monitoring_external_sg.id}"
-  cidr_blocks       = ["0.0.0.0/0"]
-}
-
-resource "aws_security_group_rule" "allow_prometheus_access_alertmanager" {
```

Comment:
I'm not that good at understanding all these security groups. Could you explain for me why we are able to remove this security group rule please :)

Reply:
This is a rule that made through from the way that we previously had prometheus configured. We had more components which required more rules. We now mainly utilise virtual hosts through the LB.

Reply:
There is a bunch of them so I believe it is worth reviewing and clean up as part of a card.

Reply:
can you explain clearly and specifically:

 - what the security group did
 - why it is not needed

When you say "we had more components which required more rules" - which components? Which rules?  What was the specific change that made this rule no longer necessary?

Reply:
We use to run prometheus and alert manager in fornt of an ALB
with an Nginx server with each service. There where multiple targets
groups which exposed alertmanager to the external facing ALB. This
meant that we needed alot of security group rules in order to
permit access to external facing ALB from the public. We also needed
internal security groups to ensure that communciation was occuring.

We have now added a signle nginx service that enables routing based
host header, including similar rules to ALB's. This means
we now no longer need to many rules.